maestrowf/datastructures/core/executiongraph.py

```diff
@@ -65,6 +65,19 @@ def __init__(self, workspace, step, **kwargs):
         self._end_time = None
         self.status = kwargs.get("status", State.INITIALIZED)
 
+        # Parameter info
+        self._params = None
+
+    def add_params(self, params):
```

Comment:
Documenting `params` appears to be missing.

Reply:
Got both missing instances of this; also patched up Combo's param values doctstring while tracing what the type of this critter was.